Cargo.toml

```
@@ -1,37 +1,54 @@
 [workspace]
 resolver = "2"
 members = [
-  "fendermint/abci",
-  "fendermint/app",
-  "fendermint/app/options",
-  "fendermint/crypto",
-  "fendermint/app/settings",
-  "fendermint/eth/*",
-  "fendermint/rocksdb",
-  "fendermint/rpc",
-  "fendermint/storage",
-  "fendermint/testing",
-  "fendermint/testing/*-test",
-  "fendermint/vm/*",
+  # contracts
+  "contracts/binding",
+  # fvm-utils,
+  "fvm-utils/primitives",
+  "fvm-utils/runtime",
+  # ipc
+  "ipc/ipc/cli",
+  "ipc/ipc/identity",
+  "ipc/ipc/provider",
+  "ipc/ipc/sdk",
+  # Tests seem to be obsolete (using IPC agent):
+  # "ipc/ipc/testing/*",
```

We should remove them. We were waiting for M2.5 to deprecate them, and now we can. I actually removed them from my monorepo PR already. 
```suggestion
  # "ipc/ipc/testing/*",
```